File: app/src/main/java/com/example/myapplication/activities/ManageOrders.java
package com.example.myapplication.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.myapplication.databinding.ActivityManageOrdersBinding;

public class ManageOrders extends AppCompatActivity {
private ActivityManageOrdersBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityManageOrdersBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        getSupportActionBar().hide();

    }
}

File: app/src/main/java/com/example/myapplication/activities/ManageOrders.kt
package com.example.myapplication.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.NavController
import androidx.navigation.fragment.NavHostFragment
import com.example.myapplication.R
import com.example.myapplication.databinding.ActivityManageOrdersBinding

class ManageOrders : AppCompatActivity() {
    private var binding: ActivityManageOrdersBinding? = null
    private lateinit var navController: NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityManageOrdersBinding.inflate(
            layoutInflater
        )
        setContentView(binding!!.root)

        val navHostFragment = supportFragmentManager.findFragmentById(R.id.orderNavHostFragment) as NavHostFragment
        navController = navHostFragment.navController
    }

    override fun onSupportNavigateUp(): Boolean {
        return super.onSupportNavigateUp() || navController.navigateUp()
    }
}